### PR TITLE
chore: add showcase test for server-side with gRPC transport

### PR DESCRIPTION
**Pull request by mpeddada1**
Thank you for opening a Pull Request! For general contributing guidelines, please refer to [contributing guide](https://github.com/googleapis/gapic-generator-java/blob/main/CONTRIBUTING.md)

Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/gapic-generator-java/issues/new/choose) before writing your code! That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

File: showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ITServerSideStreaming {
+
+  private static EchoClient grpcClient;
```
Comment (burkedavison) [inline showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java:44]: Static --> Non-static

**Review by burkedavison — APPROVED** [review]

File: showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    String content = "The rain in Spain stays mainly on the plain!";
+    ServerStream<EchoResponse> responseStream =
+        grpcClient.expandCallable().call(ExpandRequest.newBuilder().setContent(content).build());
+    ArrayList<String> responses = new ArrayList<>();
+    for (EchoResponse response : responseStream) {
+      responses.add(response.getContent());
+    }
+
+    assertThat(responses)
+        .containsExactlyElementsIn(
```
Comment (lqiu96) [inline showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java:76]: nit: add `[inOrder](https://truth.dev/api/0.25/com/google/common/truth/Ordered.html#inOrder())()`

File: showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java (hunk 1 above)
Comment (mpeddada1) [reply showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITServerSideStreaming.java]: Thank you! Done.

**Review by lqiu96 — APPROVED** [review]
Thanks @mpeddada1! Will add a HttpJson version.

**Pull request merged**